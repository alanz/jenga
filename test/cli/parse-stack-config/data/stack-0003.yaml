flags: {}

extra-package-dbs: []

packages:
  - location: wibble

resolver: lts-9.0

nix:
    packages: [gmp,git]
