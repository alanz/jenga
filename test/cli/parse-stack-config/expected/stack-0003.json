{
    "packages": [
        {
            "location": "wibble"
        }
    ],
    "extra-deps": [],
    "resolver": "lts-9.0"
}
